### PR TITLE
🐛 Check if managed cluster is created during join

### PR DESCRIPTION
**Pull request by qiujian16**
This is to check if managed cluster is created
when using --wait flag with join command, using
the bootstrap secret

[comment]: # ( Copyright Contributors to the Open Cluster Management project )

## Related issue(s)

Fixes https://github.com/open-cluster-management-io/clusteradm/issues/410

### CODE REVIEW

**Comment by mikeshng** [comment]
CC @nirs

File: pkg/cmd/join/exec.go [hunk 1]
```diff
@@ -455,6 +462,51 @@ func checkIfRegistrationOperatorAvailable(f util.Factory) (bool, error) {
...
+	if err != nil {
+		return fmt.Errorf("failed to create rest config: %v", err)
+	}
+	clusterClient, err := clusterclient.NewForConfig(restConfig)
+	if err != nil {
+		return fmt.Errorf("failed to create client: %v", err)
+	}
+
+	phase := &atomic.Value{}
+	phase.Store("")
```
Comment (zhujian7) [inline pkg/cmd/join/exec.go:478]: what is the `phase.Store("")` used for?
we did not print any status

**Pull request merged**